File: C language/fibonacci.c
#include<stdio.h>
int printFibonacciSeries(int n ){
    // n is total number of terms 
    if(n==0){
        return 0 ;
    }if(n==1){
        return 1 ;
    }
    int fib = printFibonacciSeries(n-1)+printFibonacciSeries(n-2);
    return fib;
}
int main (){
    int n ;
    printf("Enter the number of terms: ");
    scanf("%d",&n);
    printf("The fibonacci series is : ");
    for(int i = 0; i < n; i++) {
    printf("%d", printFibonacciSeries(i));
    if (i < n - 1) {
        printf(" , ");
    }
}
    return 0;
}

File: java/fibonacci.java
import java.util.Scanner;
class fibonacci {

    public static void printFibonacciSeries(int n){
        // Initialize first two terms
        int first = 0, second = 1;

        System.out.println("Fibonacci Series:");
        for (int i = 1; i <= n; i++) {
            System.out.print(first + " ,");

            // Calculate next term
            int next = first + second;
            first = second;
            second = next;
        }

    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number:");
        int n = sc.nextInt();
        printFibonacciSeries(n);
        sc.close();
    }
}